packages/web/src/public/locales/en/translate.json

```diff
@@ -1,9 +1,11 @@
 {
+  "approve_tooltip": "Approve (Enter)",
   "approved": "Approved",
   "interlinear": "Interlinear",
   "next_verse": "Next Verse",
   "previous_verse": "Previous Verse",
   "saving": "saving",
   "select_verse": "Select verse",
-  "strongs": "Strongs"
+  "strongs": "Strongs",
+  "unapprove_tooltip": "Unapprove (Esc)"
```

Comment:
Should we use a different term? "Unapprove" isn't a real word, so it doesn't translate well.

Reply:
Let's use "Revoke" instead, that's really what this is